apps/dash-3d-image-partitioning/assets/howto.md

```diff
@@ -0,0 +1,35 @@
+## What this app can do
+This dash app allows you to annotate automatically segmented brain regions. 
+You might, for example, want to annotate an area of hypoperfusion in an acutely infarcted brain and then download this annotation for further use. 
```

Comment:
Is there any way to say this (hypoperfusion) in a less technical way? I'm sure this is great for brain specialists but we want the text to be understandable by all flavors of scientists.

Reply:
Agreed. I was thinking we could explain both what is happening in the image and how you might use the app, in one sentence. Since the image is from a tumour database, I think we could just say: "Here you can see a brain with a brain tumour. You may want to annotate the area of the brain with the tumour and then download this annotation for future use". That should be more accurate and less technical I think